ErsatzTV.Infrastructure/Plex/PlexServerApiClient.cs

```diff
@@ -52,10 +52,10 @@ public async Task<bool> Ping(PlexConnection connection, PlexServerAuthToken toke
                 await service.GetLibraries(token.AuthToken).Map(r => r.MediaContainer.Directory);
             return directory
                 // .Filter(l => l.Hidden == 0)
-                .Filter(l => l.Type.ToLowerInvariant() is "movie" or "show")
-                .Filter(
-                    l => l.Type.ToLowerInvariant() is not "movie" ||
-                         (l.Agent ?? string.Empty).ToLowerInvariant() is not "com.plexapp.agents.none")
+                //.Filter(l => l.Type.ToLowerInvariant() is "movie" or "show")
```

Comment:
This needs to stay, as ETV still only supports "movie" and "show" libraries from Plex. Plex's other videos are "movie" libraries with a metadata agent of `com.plexapp.agents.none` (see the filter below and https://github.com/ErsatzTV/ErsatzTV/pull/801.

I would update the `Option<PlexLibrary> Project(PlexLibraryResponse response)` method to return the appropriate library kind based on the metadata agent.

Reply:
Yes, it is totally understable :) I'm not enough qualified with C# to easily complete this one. I will try to look deeper to update the code as you are advising.

Reply:
Ok, understood for the filter
![image](https://github.com/ErsatzTV/ErsatzTV/assets/1375001/efc01372-6913-40d5-a0e3-79a70d9ad268)